new-content/events/summer-school-2021.yaml

```diff
@@ -13,6 +13,8 @@ header:
   cta:
     label: 'Learn more'
     url: 'https://qiskit.org'
```

Comment:
Ask @JRussellHuffman if this link should have an arrow. If the answer is "yes", replace this url with "/"